components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/store/InMemoryBasedTemplateManager.java

```diff
@@ -39,7 +39,7 @@
  * This class have support for get, list, exists operations and any invocations to modification operations
  * throws {@link UnsupportedOperationException}.
  * This class expected to be use only with conjunction with another {@link TemplatePersistenceManager} implementation
- * which supports full CRUD operations, hence {@link DefaultTemplateManager} provides that aggregation using this as a
+ * which supports full CRUD operations, hence {@link UnifiedTemplateManager} provides that aggregation using this as a
  * fallback provider.
  */
 public class InMemoryBasedTemplateManager implements TemplatePersistenceManager {
```

Comment:
Renaming this to SystemDefaultTemplateManager would make much sense IMO.

Reply:
Agreed.. I also thought InMemoryBasedTemplateManager name only gives the unnecessary implementation detail yet not describe the duty of this class.. Will fix this.

Reply:
Fixed with https://github.com/wso2-extensions/identity-event-handler-notification/pull/257/commits/8cf3bf4c1b6e6867163740fa63bae3a87a3c43eb